src/contour/CMakeLists.txt

```diff
@@ -110,7 +116,11 @@ if(Boost_FILESYSTEM_FOUND)
 endif()
 
 if(CONTOUR_FRONTEND_GUI)
-    target_link_libraries(contour contour_frontend_opengl terminal_renderer Qt5::Gui Qt5::Widgets Qt5::Network)
+    if(CONTOUR_BUILD_WITH_QT6)
+        target_link_libraries(contour contour_frontend_opengl terminal_renderer Qt6::Core Qt6::Network Qt6::OpenGL Qt6::OpenGLWidgets Qt6::Widgets)
```

Comment:
`contour_frontend_opengl terminal_renderer` should be moved out of this if-condition and only qt-related libraries should be added here (or into a variable that is later on fed to `target_link_libraries()`.